src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/Descriptor.php

```diff
@@ -132,6 +135,11 @@ protected function write(string $content, bool $decorated = false)
      */
     abstract protected function describeContainerServices(ContainerBuilder $builder, array $options = []);
 
+    /**
+     * Describes container deprecations.
+     */
+    abstract protected function describeContainerDeprecations(ContainerBuilder $builder, array $options = []);
```

Comment:
Missing `: void` return type?

Reply:
The others `describeContainerXXX` already defined in this file does not declare a `void` return type. I would prefer to remain consistent with the whole file.

Reply:
But you declared it in implementations. So it should be consistent with the interface IMHO.

Reply:
Oh you are right, I did not see that. The method declarations was already added in Simperfit commit. Would you prefer `void` return type on the interface, or I remove it from the implementations ?

Reply:
Let's add it. It's accurate and does not restrict anything anyway as this class is internal.

Reply:
Ok done :)